zipline/pipeline/expression.py

```diff
@@ -328,7 +330,12 @@ def __repr__(self):
 
     def short_repr(self):
         """Short repr to use when rendering Pipeline graphs."""
-        return "Expression: {expr}".format(
+        final = self._expr
+        num_list = re.findall(r"[-+]?\d*\.\d+", self._expr)
+        for i in num_list:
+            final = final.replace(i, '%.2E' % Decimal(i))
```

Comment:
small tip: you can get access to the `%` formatting behavior for a single object with the `format` builtin:

```python
In [6]: format(Decimal(1), '.2E')
Out[6]: '1.00E+0'
```